homeassistant/components/media_player/nadtcp.py

```diff
@@ -0,0 +1,182 @@
+"""
+Support for NAD digital amplifiers which can be remote controlled via tcp/ip.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/media_player.nadtcp/
+"""
+import logging
+import voluptuous as vol
+from homeassistant.components.media_player import (
+    SUPPORT_VOLUME_SET,
+    SUPPORT_VOLUME_MUTE, SUPPORT_TURN_ON, SUPPORT_TURN_OFF,
+    SUPPORT_VOLUME_STEP, SUPPORT_SELECT_SOURCE, MediaPlayerDevice,
+    PLATFORM_SCHEMA)
+from homeassistant.const import (
+    CONF_NAME, STATE_OFF, STATE_ON)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['https://github.com/joopert/nad_receiver/archive/'
```

Comment:
Can we use the pip version here? https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/media_player/nad.py#L20